models/components/components.go

```diff
@@ -0,0 +1,61 @@
+package components
+
+import (
+	"log"
+	"sync"
+
+	"github.com/opencontrol/compliance-masonry/models/components/versions/base"
+)
+
+// Components struct is a thread-safe structure mapping for components
+type Components struct {
+	mapping map[string]base.Component
```

Comment:
Not a blocker (since this existed before this pull request), but can we call this something more informative, like `componentsByKey`?